app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+## class Like
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Please kindly ensure that this references the user class.

Hint:
`belongs_to :author, class_name: 'User'`